react-app/src/History.js

```diff
@@ -1,19 +1,76 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { StatusBar } from "expo-status-bar";
+import { StyleSheet, Text, View } from "react-native";
+import { Calendar } from "react-native-calendars";
 
 export default function History() {
+  // marked the days
+  const marked = {
+    "2023-02-18": { selected: true },
+    "2023-02-19": { selected: true },
+    "2023-02-23": { selected: true },
+    "2023-03-12": { selected: true },
+    "2023-03-14": { selected: true },
+  };
   return (
     <View style={styles.container}>
       <StatusBar style="auto" />
+      <View style={styles.calendar}>
+        {/* Calendar */}
+        <Calendar markedDates={marked} />
+        {/* Text of the record */}
+        <Text style={styles.title}>Records</Text>
+        <View style={styles.recordInfo}>
+          <Text style={styles.recordDate}>2023年2月23日</Text>
+          <Text style={styles.recordText}>Standard stretching</Text>
+        </View>
+      </View>
     </View>
   );
 }
 
+// Css
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: '#fff',
-    alignItems: 'center',
-    justifyContent: 'center',
+    backgroundColor: "#fff",
+    alignItems: "center",
+    justifyContent: "top",
```

Comment:
This line will crash android. Please propose a solution

Reply:
I think it shoud be `justifyContent: "flex-start"`

Reply:
@huishi329 
you're right! 
and I checked that the both works